src/components/app/App.jsx

```diff
@@ -1,25 +1,57 @@
-import React from 'react'
-import { Route } from 'react-router-dom'
+import React, { Component } from 'react'
+import { Redirect, Route, Switch } from 'react-router-dom'
+import { ConnectedRouter } from 'react-router-redux'
 import { Layout } from 'antd'
+import moment from 'moment'
```

Comment:
Qu'est-ce que c'est ?

Reply:
A lib to handle with simplicity the date (ask if a date is after another or add time to a date without checking the format). It's far better than the basic `Data` type of js.